knexfile.js

```diff
@@ -2,37 +2,54 @@
 require('dotenv').config();
 const pg = require('pg');
 
-pg.defaults.ssl = false; // this needs to be false in development, true when using heroku db
+pg.defaults.ssl = true; // this needs to be false in development, true when using heroku db
 
 module.exports = {
 	development: {
 		client: 'pg',
 		connection: {
 			host: '127.0.0.1',
-			user: 'justin',
+			user: 'cotikor',
```

Comment:
Is justin's name needed for access to Heroku at all? Like is that equal to the heroku username? Just double checking.